docker/dockerfile-cli

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && \
+    apt-get install -y cmake build-essential gcc g++ git && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN git clone --recursive https://github.com/Microsoft/LightGBM && \
```

Comment:
using latest tag. @guolinke , maybe we can maintain an "latest&stable" tag for the release.

Reply:
Which tag do you think is a better choice than master, @chivee ?

An alternative could be to build the container based on the current git working tree and skip the cloning entirely.

Reply:
let's using [stable](https://github.com/Microsoft/LightGBM/releases/tag/stable) tag for the docker release, this tag should pointer to the latest release/tagged version.